pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -502,3 +512,85 @@ func checkForPodSecondaryNetworkAttachement(pod *corev1.Pod) (string, bool) {
 		return netObj, false
 	}
 }
+
+// initializeSecondaryInterfaceStore restores secondary interfaceStore when agent restarts.
+func (pc *PodController) initializeSecondaryInterfaceStore() error {
+	if pc.ovsBridgeClient == nil {
```

Comment:
is this actually possible? it is strange to just silently do nothing when `ovsBridgeClient` is nil. If this is not a supported case, IMO it is better to remove the check.

Reply:
I added this to handle :https://github.com/antrea-io/antrea/blob/main/pkg/agent/secondarynetwork/podwatch/controller_test.go#L223, in  TestPodControllerRun() 
OVSBridgeClient) and  primaryInterface store are nil.
```
podController, _ := NewPodController(
		client,
		netdefclient,
		informerFactory.Core().V1().Pods().Informer(),
		nil, nil, nil) 
```

Reply:
It's usually not a good idea to modify your logic specifically to accommodate for unit tests, it would make more sense to modify the tests instead